index.bs

```diff
@@ -6895,8 +6895,8 @@ to grow organically as needed.
      otherwise. If this throws an exception |e|, return [=a promise rejected with=] |e|.
   1. If |result| is a {{Promise}}, then return |result|.
   1. Return [=a promise resolved with=] undefined.
- 1. Let |cancelAlgorithmWrapper| be an algorithm that runs these steps:
-  1. Let |result| be the result of running |cancelAlgorithm|, if |cancelAlgorithm| was given, or
+ 1. Let |cancelAlgorithmWrapper| be an algorithm that runs these steps given |reason|:
+  1. Let |result| be the result of running |cancelAlgorithm| given |reason|, if |cancelAlgorithm| was given, or
```

Comment:
Rewrap to 100 chars here and below

Reply:
[CONTRIBUTING.md in this repository](https://github.com/whatwg/streams/blob/main/CONTRIBUTING.md#:~:text=Spec%20editorial%20conventions-,Wrap%20lines%20to%20120%20columns.,-\(This%20is%20not) says the line limit in this specification is 120 chars, unlike other WHATWG repositories.

Should I comply with that right now, or do you still want this wrapped at 100?

Reply:
It looks like CONTRIBUTING.md is outdated; 100 would be best, please!

Reply:
Done.